release-conf.yaml

```diff
@@ -1,7 +1,3 @@
-# list of major python versions that release-bot will build separate wheels for
-python_versions:
-  - 3
```

Comment:
We need this until new-release bot (with https://github.com/user-cont/release-bot/pull/147) is released